src/lints/inherent_method_must_use_added.ron

```diff
@@ -0,0 +1,80 @@
+SemverQuery(
+    id: "inherent_method_must_use_added",
+    human_readable_name: "inherent method #[must_use] added",
+    description: "An inherent method has been marked with #[must_use].",
+    required_update: Minor,
+
+    // TODO: Change the reference link to point to the cargo semver reference
+    //       once it has a section on attribute #[must_use].
+    reference_link: Some("https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute"),
+    query: r#"
+    {
+        CrateDiff {
+            current {
+                item {
+                    ... on ImplOwner {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        name @tag @output
+                        owner_type: __typename @tag
```

Comment:
```suggestion
                        owner_type: __typename @tag @output
```
Just for the sake of easier debugging, if we ever need it.

Trustfall guarantees that `@output` directives are processed only after all filtering / type coercions etc. is done, so adding `@output` has zero performance cost on results that aren't actually returned. Any results that are returned are lints, so there are almost certainly not even remotely enough of them to make any noticeable performance difference whatsoever.

So it's probably a good idea just for our debugging convenience to always `@output` values that are used in with a `@tag`.